views/event-detail.html

```diff
@@ -0,0 +1,34 @@
+{% extends "layout.html" %}
+
+{% block mainContent %}
+  <div>
+    <p>What: {{event.title}}</p>
+    <p>When: {{event.date}}</p>
```

Comment:
This date should be formatted better, using the `time` tag, setting the `datetime` attribute, and including a formatted date using javascript (e.g. Feb 20, 2015, or something like that).